server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/node/metedata/write/AlterTimeSeriesNode.java

```diff
@@ -263,8 +213,49 @@ public <R, C> R accept(PlanVisitor<R, C> visitor, C schemaRegion) {
 
   @Override
   protected void serializeAttributes(ByteBuffer byteBuffer) {
-    throw new NotImplementedException(
-        "serializeAttributes of AlterTimeSeriesNode is not implemented");
+    byteBuffer.putShort((short) PlanNodeType.ALTER_TIME_SERIES.ordinal());
```

Comment:
may be it is better to use `PlanNodeType#serialize`,   and why use `ordinal()` other than `nodeType`?

Reply:
good idea! fixed.